api/src/wfl/service/rawls.clj

```diff
@@ -37,14 +39,62 @@
    (create-snapshot-reference workspace snapshot-id name "")))
 
 (defn get-snapshot-reference
-  "Return the snapshot reference in fully-qualified Terra WORKSPACE with REFERENCE-ID."
+  "Return the snapshot reference in `workspace` with `reference-id`."
   [workspace reference-id]
-  (get-workspace-json workspace "snapshots" reference-id))
+  (get-workspace-json workspace snapshot-endpoint reference-id))
+
+(defn get-snapshot-references
+  "Lazily returns the snapshot references in `workspace`
+  with `limit` resources per page (default: 100)."
```

Comment:
"Return a lazy sequence of snapshot references in …"
Should the caller know or care about the page size?

Reply:
I guess we don't need to expose it, but I did find it helpful in testing to confirm my understanding of the pagination!